components/user-mgt/org.wso2.carbon.user.mgt.workflow/src/main/java/org/wso2/carbon/user/mgt/workflow/userstore/UpdateRoleNameWFRequestHandler.java

```diff
@@ -178,14 +176,10 @@ public boolean isValidOperation(Entity[] entities) throws WorkflowException {
 
         WorkflowService workflowService = IdentityWorkflowDataHolder.getInstance().getWorkflowService();
         for (int i = 0; i < entities.length; i++) {
-            try {
-                if (entities[i].getEntityType() == UserStoreWFConstants.ENTITY_TYPE_ROLE && workflowService
+            if (entities[i].getEntityType() == UserStoreWFConstants.ENTITY_TYPE_ROLE && workflowService
```

Comment:
Isn't that we have to use 'equals()' instead '==' as we are doing a object content comparison?